docs/DeveloperGuide.adoc

```diff
@@ -782,13 +782,15 @@ See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-
 
 *Target user profile*:
 
-* has a need to manage a significant number of contacts
+* <<member>> of a <<club>>
```

Comment:
Instead of using placeholders like <<member>> and <<club>> , we can just say that the target user(s) of the app is/are 'Members of Student Organizations across Campus'

Reply:
Oh, those aren't placeholders. I was planning to link those words to their definitions in the glossary. I'll fix the link format.